components/dashboard/src/Menu.tsx

```diff
@@ -260,33 +253,23 @@ export default function Menu() {
     const onFeedbackFormClose = () => {
         setFeedbackFormVisible(false);
     };
+    const isTeamLevelActive = !projectSlug && !isWorkspacesUI && !isAdminUI && teamOrUserSlug;
     const renderTeamMenu = () => {
         const classes =
             "flex h-full text-base py-0 " +
-            (!projectSlug && !isWorkspacesUI && !isAdminUI && teamOrUserSlug
-                ? "text-gray-50 bg-gray-800 dark:text-gray-900 dark:bg-gray-50 border-gray-700"
-                : "text-gray-500 bg-gray-50 dark:bg-gray-800 dark:text-gray-400 hover:bg-gray-100 dark:hover:bg-gray-700  dark:border-gray-700");
+            (isTeamLevelActive
+                ? "text-gray-50  bg-gray-800 dark:bg-gray-50  dark:text-gray-900 border-gray-700 dark:border-gray-200"
+                : "text-gray-500 bg-gray-50  dark:bg-gray-800 dark:text-gray-400 hover:bg-gray-100 dark:hover:bg-gray-700 dark:border-gray-700");
         return (
             <div className="flex p-1 pl-3">
-                {projectSlug && (
-                    <Link to={team ? `/t/${team.slug}/projects` : `/projects`}>
-                        <span
-                            className={`${classes} rounded-tl-2xl rounded-bl-2xl  dark:border-gray-700 border-r pl-3 pr-2 py-1 bg-gray-50  font-semibold`}
-                        >
-                            {team?.name || userFullName}
-                        </span>
-                    </Link>
-                )}
-                {!projectSlug && (
-                    <Link to={team ? `/t/${team.slug}/projects` : `/projects`}>
-                        <span
-                            className={`${classes} rounded-tl-2xl rounded-bl-2xl  dark:border-gray-200 border-r pl-3 pr-2 py-1 bg-gray-50  font-semibold`}
-                        >
-                            {team?.name || userFullName}
-                        </span>
-                    </Link>
-                )}
-                <div className={`${classes} rounded-tr-2xl rounded-br-2xl dark:border-gray-700  px-1 bg-gray-50`}>
+                <Link to={team ? `/t/${team.slug}/projects` : `/projects`}>
+                    <span
+                        className={`${classes} rounded-tl-2xl rounded-bl-2xl border-r pl-3 pr-2 py-1 bg-gray-50  font-semibold`}
+                    >
+                        {teams?.find((t) => t.slug === getSelectedTeamSlug())?.name || userFullName}
```

Comment:
**issue**: Going from workspaces back to the team-scope always goes back to the personal account projects, regardless what the selected team scope is. This should get you back to the projects of the last selected team, right?